lib/elsa/group/manager.ex

```diff
@@ -118,28 +118,6 @@ defmodule Elsa.Group.Manager do
     GenServer.call(pid, :revoke_assignments, 30_000)
   end
 
-  @doc """
```

Comment:
Should we add a defdelegate to Acknowledger to keep backwards compatibility?

Reply:
Didn’t realize that was there. Sure thing.

Reply:
Never mind; I just hadn't seen delegates outside the context of a top-level module passing down to deeper modules. Understood and added.